fudgeo/geopkg.py

```diff
@@ -517,7 +517,7 @@ def escaped_name(self) -> str:
         Escaped Name, only adds quotes if needed
         """
         name = self.name
-        if name.upper() in KEYWORDS:
+        if name.upper() in KEYWORDS or any((c in set('.-/[]()')) for c in name):
```

Comment:
thanks for this addition, even though this is not a part of the code that requires speed we still have a preference towards faster approaches for checking such as:

```python
from re import match
...
if name.upper() in KEYWORDS or match(r'\W', name):
```

of faster again (at least in 3.11)

```python
if name.upper() in KEYWORDS or set(name).intersection('.-/[]() '):
```

Reply:
Sure! There's probably a dozen ways to do that check ;-)
Gotta say I haven't looked into the 3.11 new ways of matching, even though I'm using 3.11 now. 
As long as there's a way to allow those special chars in column names, I'm happy. Feel free to reject the PR and use the fastest way of checking.